stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -82,15 +84,28 @@ public fun Messages(
     onMessagesStartReached: () -> Unit,
     onLastVisibleMessageChanged: (Message) -> Unit,
     onScrolledToBottom: () -> Unit,
+    onMessagesEndReached: (String) -> Unit,
```

Comment:
These are a breaking change, we should duplicate this function without these params and then just kind of disable the new functionality in older component alternatives, while deprecating it